src/Configuration/GraphlintConfigBuilder.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Worksome\Graphlint\Configuration;
+
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+use Worksome\Graphlint\GraphlintSet;
+use Worksome\Graphlint\Inspections\IgnoreByNameSuppressorInspection;
+use Worksome\Graphlint\Inspections\Inspection;
+
+final class GraphlintConfigBuilder
+{
+    /** @var array<GraphlintSet> */
+    private array $sets = [];
+
+    /** @var array<class-string<Inspection>> */
+    private array $inspections = [];
+
+    /** @var array<string> */
+    private array $ignored = [];
+
+    public function __invoke(ContainerConfigurator $config): void
+    {
+        $services = $config->services();
+
+        foreach ($this->sets as $set) {
+            $config->import($set->value);
+        }
+
+        foreach ($this->inspections as $inspection) {
+            $services->set($inspection)->tag(Visitor::COMPILED);
```

Comment:
Currently just tagging all inspections as being for the compiled schema, as we don't support original yet.